### ISSUE TITLE
IndentLine doesn't working (or loading) at all

### ISSUE THREAD

**Issue by pagubg** [Yggdroot/indentLine#148]
my vimrc:
==========================
:color desertexecute pathogen#infect()
if has("autocmd")
        autocmd BufRead *.sql set filetype=mysql      
        autocmd BufRead *.test set filetype=mysql
endif
" filetype plugin on
filetype plugin indent on

" F4 - toggle indent guides
nmap <F4> :IndentLinesToggle<CR>
imap <F4> <C-o>:IndentLinesToggle<CR>

==========================
vim version: VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Jan  2 2014 19:39:32)
OS Version: 3.19.0-33-generic #38~14.04.1-Ubuntu 14.04.4 LTS

Problem Description:
When I open any type of file (tried with php, C, shell, CPP, etc), I can't see any indents.
Script seems to be loaded correctly, at least :scriptnames shows
` 24: ~/.vim/bundle/indentLine/after/plugin/indentLine.vim`

**Comment by charlieWasp** [comment]
Same problem here, installed via Vundle, but even can't see `indentLine` in `scriptnames` output

**Comment by claygorman** [comment]
+1

**Comment by Yggdroot** [comment]
similar with issue #59

**Comment by atsepkov** [comment]
@Yggdroot I too am seeing this issue, but it doesn't seem in any way related to #59. That issue is about configuring installed copy of indentLine to activate. This issue seems to be that despite being installed by package manager, indentLine behaves as if it was never installed. In fact, none of the commands associated with the plugin are even picked up by the editor command line. `scriptnames` does not show any trance of indentLine.

I'm using neovim by the way, but other neovim users seem to be able to get the plugin to work.

**Comment by Yggdroot** [comment]
Please refer to issue #203.

**Comment by atsepkov** [comment]
Ok, I figured out what my issue was. The plugin was apparently loading but 2 other things were happening:
A: the color scheme I used overrode Conceal to be invisible
B: when I was trying to explicitly set the color via g:indentLine_color_term, it was being ignored because I had termguicolors set, which told my terminal to use the same colors for terminal vim as for gvim. When I set g:indentLine_color_gui to a different color, I am able to see the indent markers.
